drop-in/src/main/res/values/strings.xml

```diff
@@ -12,15 +12,16 @@
     <string name="action_failed">There was an error while processing your payment. Please try again later.</string>
     <string name="time_out_error">Request timed out. Please try again.</string>
     <string name="unknown_error">An unknown error occurred</string>
-    <string name="store_payment_methods_header">Your payment methods</string>
-    <string name="other_payment_methods">Select other method</string>
+    <string name="store_payment_methods_header">Stored</string>
```

Comment:
Let's name these strings something more specific to avoid conflicting with the merchants' own strings. Maybe prefix them with `checkout_` to follow the pattern?

Reply:
For this PR I only updated the relevant ones, for the remaining ones we can do them in a separate PR to keep this one focused on headers.

Reply:
We can't update any string that's already released anyway or else it's a breaking change, but maybe we should make sure all the strings that meal vouchers (or related PRs) recently introduced are prefixed